cmake.deps/cmake/BuildTreesitterParsers.cmake

```
@@ -1,14 +1,29 @@
+# Helper function to download treesitter parsers
+#
+# Single value arguments:
+# LANG        - Parser language
+# CMAKE_FILE  - Cmake file to build the parser with. Defaults to
+#               TreesitterParserCMakeLists.txt.
 function(BuildTSParser)
   cmake_parse_arguments(TS
     ""
-    "LANG;URL;SHA256;CMAKE_FILE"
+    "LANG;CMAKE_FILE"
     ""
     ${ARGN})
 
+  if(NOT TS_CMAKE_FILE)
+    set(TS_CMAKE_FILE TreesitterParserCMakeLists.txt)
+  endif()
+
   set(NAME treesitter-${TS_LANG})
```

```suggestion
  set(NAME tree-sitter-${TS_LANG})
```
Upstream consistently uses `tree-sitter`; the spelling `treesitter` is a Neovim idiosyncrasy.

Specifically, the actual repos are all named `tree-sitter-<lang>`.